app/services/import_references_from_csv.rb

```diff
@@ -0,0 +1,59 @@
+require 'csv'
+
+class ImportReferencesFromCsv
+  def self.call(csv_file:)
+    outcomes = []
+    CSV.foreach(csv_file) do |row|
```

Comment:
This code might be more readable using the `headers: true` option for `CSV.open` (https://stackoverflow.com/a/3717622), which means there's no need for numeric indexes in `process_row`.

Reply:
@duncanjbrown I started with headers, but switched. There's risk in using headers given the way Google Forms spreadsheets work. The labels for the columns might change. The order of the columns will not (new fields get appended to the end). The column headings can also get quite long.